lib/xenops_server.ml

```diff
@@ -1597,6 +1602,7 @@ and perform ?subtask ?result (op: operation) (t: Xenops_task.t) : unit =
 			debug "VM.receive_memory %s power_state = %s" id (state.Vm.power_state |> rpc_of_power_state |> Jsonrpc.to_string);*)
 
 			(* set up the destination domain *)
+			debug "VM.receive_memory creating domain and restoring VIFs";
```

Comment:
This should be ok, we have the events in the queue and the related task ids that allow for easy tracing of the vm.